jekyll/_cci2/automate-llm-evaluation-testing-with-the-circleci-evals-orb.adoc

```diff
@@ -0,0 +1,252 @@
+
+[source ,yaml]
+----
+jobs:
+  eval-test:
+    docker: 
+      - image: cimg/python:stable
+    steps:
+      # Add an evaluation step
+      # - run: <your eval step> # Run evals and output results for the evaluation test
+      - evals/test: # Invoke evaluation test
+          name: Run Evaluation Test
+          assertions: <your path to .json evaluation test suite> # Replace with your path
+          metrics: <your path to .json evaluation results> # Replace with your path
+          results: <your path to store JUnit XML evaluation test results file> # Replace with your path
+----
+
+== 3. Create evaluation test suite file
+
+* Create a `.json` file. You can choose the file location, but we suggest placing it in your code repository `.circleci` directory.
+* You can customize the filename. We suggest using `eval-test.json`, since this file represents the test suite for testing evaluation results.
+
+== 4. Define test cases
+
+* In the `.json` file you just created, define your test cases.
+* Each test case is composed of a name and an assertion. 
+* For test case names we suggest assigning a name based on the test scenario. For example, using the name of a metric: “`correctness`” or “`toxicity`.”
+* Assertions are expressed using the link:https://cel.dev/[Common Expression Language] (CEL). For more information, see the link:https://github.com/google/cel-spec/blob/master/doc/langdef.md[specification]. All values from the results `.json` file are available as global variables. Try out CEL link:https://playcel.undistro.io/[here].
+
+=== Test case examples
+
+Given metrics from `eval-results.json`:
+
+[source ,json]
+----
+{
+  "correctness": 0.99,
+  "toxicity": 0.2,
+  "labels": [
+    "CORRECT",
+    "ACCURATE",
+    "NOT HARMFUL"
+  ]
+}
+----
+
+Below are example test cases and a demonstration of link:https://cel.dev/[Common Expression Language] (CEL) usage.
+
+* For test case names we suggest assigning a name based on the test scenario. For example, using the name of an input metric: “`correctness`” or “`toxicity`.”
+
+[source ,json]
+----
+{
+  "correctness": "correctness > 0.9"
+  "toxicity": "toxicity < 0.01"
+  "labels": “labels[0] == \”CORRECT\””
+}
+----
+
+Below displays a template, demonstrating that a test case is composed of a name and an assertion:
+
+[source ,json]
+----
+{
+  "yourTestCaseName": "<your assertion>",
+}
+----
+
+== 5. Add eval job to a workflow
+
+Define a new workflow, or use an existing one. Add your newly defined job to the workflow. Under your job, specify the relevant contexts needed to run the tasks in your job.
+
+[source ,yaml]
+----
+workflows:
+  build-test-eval-workflow:
+    jobs:
+      - build-test-eval:
+          context:
+            - <your OpenAI context> # Replace with your context
+----
+
+== 6. Examples
```

Comment:
I think this examples section isn't a "step", but more supplementary examples, so can be moved to the end of the content after the Review results step.